modules/eks/default.auto.tfvars

```diff
@@ -14,7 +14,7 @@ cluster_endpoint_private_access = true
 
 cluster_endpoint_public_access = true
 
-cluster_kubernetes_version = "1.17"
+cluster_kubernetes_version = "1.18"
```

Comment:
Note, bumping to version `1.18` since figured that is preferable. We can also update to `1.19` as well since that is out.